addons/l10n_hk/tests/test_l10n_hk_emv_qr.py

```diff
@@ -79,10 +84,7 @@ def l10n_hk_test_emv_qr_vals(self):
             structured_communication=self.emv_qr_invoice.payment_reference,
         )
 
-        # Check crc16 value
-        crc16 = emv_qr_vals[-1]
-        self.assertEqual(crc16, '3CA6')
-
```

Comment:
No longer tested? (I guess it's not that useful since it's hardcoded, but if it is a legal requirement it doesn't hurt to test it)

Reply:
This value will be tested in the qr_code_string anyway so I think we could skip this